buildSrc/src/main/java/Dependencies.kt

```diff
@@ -30,7 +30,7 @@ object Versions {
     const val disklrucache = "2.0.2"
     const val leakcanary = "2.4"
 
-    const val mozilla_appservices = "72.1.0"
+    const val mozilla_appservices = "72.1.0-TESTING"
```

Comment:
Change this to the appservices release with the Nimbus changes from [here](https://github.com/mozilla/nimbus-sdk/pull/97) in it.